src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -42,8 +42,6 @@
     <TargetName>aspnetcorev2</TargetName>
     <LinkIncremental>false</LinkIncremental>
     <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
-    <PlatformToolsetVersion Condition=" '$(VisualStudioVersion)' == '17.0' ">v143</PlatformToolsetVersion>
-    <PlatformToolsetVersion Condition=" '$(PlatformToolsetVersion)' == '' ">v142</PlatformToolsetVersion>
```

Comment:
@dougbu I believe you added these in https://github.com/dotnet/aspnetcore/commit/9217656f967cc6d040071df73b99cd94fddc5e84#diff-5f4ea5312e8f8d98f704201802678b117219b89f48cc8283330a652faf579749 now that we are using a static lib from runtime, we are required to use the same version they do, which is v142, so I've just removed all of these overrides since they break things, and hardcode v142 which is required in the directory.build props, do you see any problem with this?

Reply:
I get that this was trying to give users some wiggle room previously, but now that we are consuming the libnethost.lib static library from the runtime repo, we are required to compile with the same toolset they are (v142).  

Actually this is going to cause another problem in the very near future, in that we require v143 tools for our arm64 builds, so @elinor-fung do we need runtime to upgrade to v143 tools, so we can consume libnethost on arm64?

Reply:
Is the runtime repo doing CI builds exclusively on VS2019 agents❔ And, of so, how are they supporting VS1022 users❔ This isn't like other components, where the version can be controlled (at least to some extent) when choosing VS components.

Reply:
I don't really have answers to those questions, but previously we were hardcoded to v142, and that's what we need to do for things to link against the static library currently.  At least in my VS today, after hardcoding v142, what I see in VS is that it just displays it as VS2019 next to the project

Reply:
Looks like this for me and works fine in VS2022

![image](https://user-images.githubusercontent.com/6537861/152064796-197170a4-30eb-457b-8bdb-e074ab86a5d0.png)